examples/js/renderers/SVGRenderer.js

```diff
@@ -87,14 +87,45 @@ THREE.SVGRenderer = function () {
 
 	this.setPixelRatio = function () {};
 
-	this.setSize = function ( width, height ) {
+	this.getSize = function ( target ) {
+
+		if ( target === undefined ) {
+
+			console.warn( 'SVGRenderer: .getsize() now requires a Vector2 as an argument' );
+
+			target = new Vector2();
+
+		}
+
+		var width = _svgWidth;
+		var height = _svgHeight;
+		if ( width === undefined || height === undefined ) {
+
+			// If setSize has not been called, we use the bounding client rect,
+			// rather than the SVG contents rect (getBBox), to respect CSS based sizing.
+			let box = _svg.getBoundingClientRect();
+			width = width === undefined ? box.width : width;
+			height = height === undefined ? box.height : height;
+
+		}
+
+		return target.set( width, height );
+
+	};
+	
+	this.setSize = function ( width, height, updateStyle = true ) {
```

Comment:
The PR is about `getSize()`. Why do you enhance `setSize()`?  I think there is no need for introducing a third parameter.

Reply:
Sorry, meant to submit that as a separate PR... doing too many things at once.

CSS based sizing of the svg element is destroyed by setting width/height properties on the element, but setSize is still required as you need the viewbox set to prevent the contents from being off center. The updateStyle parameter was taken from setSize in WebGLRenderer which offers updateStyle for the same reason (CSS based sizing).

Reply:
Please undo this change. It can be discussed separately.